azure-pipelines/pull-request-validation/build-consumers.yml

```diff
@@ -85,12 +85,30 @@ stages:
         }
       name: echovar
       displayName: Echo branch name
+  # Check PR Label
+  - job: checkLabel
+    displayName: Check for PR Label
+    dependsOn: setupBranch
+    steps:
+    - bash: |
+        echo "Looking for label at https://api.github.com/repos/$BUILD_REPOSITORY_ID/issues/$SYSTEM_PULLREQUEST_PULLREQUESTNUMBER/labels"
+        if curl -s "https://api.github.com/repos/$BUILD_REPOSITORY_ID/issues/$SYSTEM_PULLREQUEST_PULLREQUESTNUMBER/labels" | grep '"name": "Skip-Consumers-Check"'
+        then
+          echo "##vso[task.setvariable variable=skipValidation;isOutput=true]true"
+          echo "Skip-Consumers-Check label found, skip validation"
+        fi
+      condition: eq(variables['Build.Reason'], 'PullRequest')
+      name: checkPrCILabel
   # msal
   - job: msalValidation
     displayName: MSAL
-    dependsOn: setupBranch
+    dependsOn:
```

Comment:
suggestion:
we can have two stages.. 1. to check PR label, 2. to build consumers..
that way we can apply the condition at stage level instead of each job level.

Reply:
Yeah that makes more sense

Reply:
Seems the output variable is not set when setting condition at stage level, was only able to access it when setting condition at job level